docs/source/cpp/streaming_execution.rst

```diff
@@ -86,25 +86,21 @@ Plans
 ^^^^^
 
  * A plan should have a single top-level relation.
- * The consumer is currently based on a custom build of Substrait that
-   is older than 0.1.0.  Any features added that are newer than 0.1.0 will
-   not be supported.
+ * The consumer is currently based on a version 0.20.0 of Substrait.
+   Any features added that are newer will not be supported.
+ * Due to a breaking change in 0.20.0 any Substrait plan older than 0.20.0
```

Comment:
Do we have a validation mechanism for this?

Reply:
We do check for (and reject) older plans (but I had the wrong version and have fixed that).

Reply:
Newer plans should generally be "ok".  protobuf (and Substrait) are supposed to be forwards compatible.